extensions/amp-sidebar/1.0/toolbar.js

```diff
@@ -32,6 +35,12 @@ export class Toolbar {
     /** @private {!Object} **/
     this.win_ = win;
 
+    /** @private {Element|null} */
```

Comment:
Error in the linter: ERROR - actual parameter 1 of module$src$style.setStyles does not match formal parameter